_assets/img/spritesheet.svg

```diff
@@ -167,4 +167,7 @@
   <symbol id="fa-github" viewbox="0 0 22 21">
     <path d="M22 10.763c0 2.345-.7 4.454-2.098 6.327-1.4 1.874-3.206 3.17-5.422 3.89-.257.046-.446.013-.565-.099a.553.553 0 0 1-.18-.42v-2.957c0-.906-.248-1.57-.744-1.99.544-.056 1.034-.14 1.468-.252a5.923 5.923 0 0 0 1.346-.547c.463-.252.85-.563 1.16-.932.31-.369.564-.86.76-1.471.195-.612.293-1.315.293-2.11 0-1.13-.377-2.092-1.131-2.887.353-.85.315-1.803-.115-2.858-.267-.085-.654-.033-1.16.154a7.79 7.79 0 0 0-1.318.616l-.544.337A10.375 10.375 0 0 0 11 5.199c-.945 0-1.862.122-2.75.365a12.75 12.75 0 0 0-.609-.379c-.253-.15-.651-.329-1.196-.54-.544-.21-.955-.273-1.231-.188-.42 1.055-.454 2.008-.1 2.858-.755.795-1.132 1.757-1.132 2.887 0 .795.098 1.495.293 2.103.196.607.447 1.097.752 1.471.306.374.69.687 1.153.939.463.252.912.434 1.347.547.434.112.924.196 1.468.252-.382.336-.616.817-.702 1.443-.2.094-.415.164-.645.21a4.14 4.14 0 0 1-.816.07c-.315 0-.628-.1-.938-.3-.31-.202-.575-.494-.795-.877a2.282 2.282 0 0 0-.695-.728c-.281-.187-.518-.3-.709-.337l-.286-.042c-.2 0-.34.021-.416.063-.076.042-.1.096-.071.161a.77.77 0 0 0 .129.197c.057.065.12.121.186.168l.1.07c.21.093.418.27.623.532.206.262.356.5.451.715l.144.323c.124.355.334.642.63.861.296.22.616.36.96.42.343.062.675.094.995.099.32.005.585-.012.795-.05l.33-.055c0 .355.002.77.006 1.247l.008.757a.543.543 0 0 1-.187.42c-.124.112-.315.145-.572.098-2.216-.719-4.023-2.015-5.422-3.889C.7 15.217 0 13.108 0 10.763 0 8.81.492 7.009 1.475 5.36A10.836 10.836 0 0 1 5.48 1.443C7.164.481 9.004 0 11 0s3.836.481 5.521 1.443a10.836 10.836 0 0 1 4.004 3.917C21.508 7.01 22 8.81 22 10.763z" fill="#313131"/>
   </symbol>
+  <symbol id="fa-envelope-o" viewbox="0 0 22 17">
+    <path d="M20.429 15.068V5.795c-.262.29-.545.556-.848.797-2.193 1.658-3.936 3.019-5.23 4.081-.417.346-.757.616-1.018.81-.262.192-.616.388-1.062.585-.446.197-.866.296-1.259.296h-.024c-.393 0-.813-.099-1.259-.296-.446-.197-.8-.393-1.062-.586a24.059 24.059 0 0 1-1.019-.809c-1.293-1.062-3.036-2.423-5.23-4.08a7.144 7.144 0 0 1-.847-.798v9.273c0 .105.04.195.117.272a.38.38 0 0 0 .276.115h18.072a.38.38 0 0 0 .276-.115.368.368 0 0 0 .117-.272zm0-12.69v-.295l-.007-.157-.036-.151-.068-.109-.11-.09-.172-.03H1.964a.38.38 0 0 0-.276.114.368.368 0 0 0-.117.272c0 1.352.602 2.495 1.805 3.429 1.58 1.223 3.22 2.5 4.923 3.827.05.04.192.16.43.356.237.198.425.349.565.453.139.105.32.232.546.38.225.15.432.26.62.333.188.072.364.108.528.108h.024c.164 0 .34-.036.528-.108.188-.073.395-.184.62-.333.225-.148.407-.275.546-.38.14-.104.328-.255.565-.453.238-.197.38-.316.43-.356 1.702-1.328 3.343-2.604 4.923-3.827.442-.346.853-.811 1.234-1.395.38-.583.57-1.113.57-1.587zM22 1.933v13.136c0 .531-.192.986-.577 1.365a1.908 1.908 0 0 1-1.387.567H1.964c-.54 0-1.002-.19-1.387-.567A1.845 1.845 0 0 1 0 15.068V1.932C0 1.4.192.946.577.567A1.908 1.908 0 0 1 1.964 0h18.072c.54 0 1.002.19 1.387.567.385.379.577.834.577 1.365z" fill="#000"/>
```

Comment:
Can we wrap the lines so it's easier to see how large those svgs are. It also makes it easier to see the `fill` attribute?